README.md

```diff
@@ -164,10 +164,12 @@ function requestFn(url, data) {
 
   const { request: nodeRequest } =
     url.indexOf("https://") === 0 ? require("https") : require("http");
-  const req = nodeRequest(options);
+  const req = nodeRequest(reqOptions);
 
   req.on("error", error => {
     console.error(error);
```

Comment:
Shouldn't we just raise the error object?

Reply:
Yeah, the idea is to add the ability to handle errors without going the breaking change route, if possible 😁